### PR TITLE
[ADMIN] Update publications.json

### PR DESCRIPTION
**Pull request by gocadimic**
3 ETSs added to 'publication' file:

- ETS LwM2M v1.0.2, [OMA-ETS-LightweightM2M-V1_0_2-20180815-A.zip](https://openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M-V1_0_2-20180815-A.zip)

- ETS LwM2M v1.1, [OMA-ETS-LightweightM2M-V1_1-20190912-D.pdf](https://openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M-V1_1-20190912-D.pdf), and

- ETS LwM2M v1.2, [OMA-ETS-LightweightM2M_INT-V1_2-20231003-A.pdf](https://openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M_INT-V1_2-20231003-A.pdf)

### CODE REVIEW

**Comment by gocadimic** [comment]
@jpradocueva,
Please check the updates, especially the descriptions in all three documents. I couldn't relate that with the given example, v.1.0.1.

File: publications.json [hunk 1]
```diff
@@ -14864,6 +14864,45 @@
...
+    "organization": "https://technical.openmobilealliance.org/organization-info.json",
+    "enablerName": "OMA Enabler Test Specification for LightweightM2M (LwM2M)",
+    "enablerAbbreviation": "LightweightM2M",
+    "status": "Approved",
+    "version": "V1.0.2",
+    "releaseDate": "2018-08-15",
+    "fileName": "OMA-ETS-LightweightM2M-V1_0_2-20180815-A",
+    "docType": "Enabler Test Specification",
+    "url": "https://openmobilealliance.org/release/LightweightM2M/ETS/OMA-ETS-LightweightM2M-V1_0_2-20180815-A.zip",
+    "description": "This document describes in detail available test cases for LightweightM2M as specified in OMA-TS-LightweightM2M-V1_0. The test cases are split in two categories, conformance and interoperability test cases. The conformance test cases are aimed to verify the adherence to normative requirements described in the technical specifications. The interoperability test cases are aimed to verify that implementations of the specifications work satisfactory. If either conformance or interoperability tests do not exists at the creation of the test specification this part should be marked not available.",
```
Comment (jpradocueva) [inline publications.json:14877]: @gocadimic This description is better to take it from the Introduction. We don't do Conformance test cases. Therefore, this description doesn't provide any input. The content in the Introduction session is better and provides further information.

**Review by jpradocueva — CHANGES_REQUESTED** [review]
@gocadimic, change the description to the content in the Introduction section of the document. It is more relevant as I mentioned on my inline comments.

**Review by jpradocueva — APPROVED** [review]
@gocadimic approved. Ready to merge.

**Pull request merged**